subworkflows/nf-core/fastq_create_umi_consensus_fgbio/main.nf

```diff
@@ -0,0 +1,206 @@
+//
+// Runs FGBIO tools to remove UMI tags from FASTQ reads
+// Convert them to unmapped BAM file, map them to the reference genome,
+// use the mapped information to group UMIs and generate consensus reads
+//
+
+include { BWA_INDEX                         as BWAMEM1_INDEX       } from '../../../modules/nf-core/bwa/index/main.nf'
+include { BWA_MEM                           as BWAMEM1_MEM_PRE     } from '../../../modules/nf-core/bwa/mem/main.nf'
+include { BWA_MEM                           as BWAMEM1_MEM_POST    } from '../../../modules/nf-core/bwa/mem/main.nf'
+include { BWAMEM2_INDEX                                            } from '../../../modules/nf-core/bwamem2/index/main.nf'
+include { BWAMEM2_MEM                       as BWAMEM2_MEM_PRE     } from '../../../modules/nf-core/bwamem2/mem/main.nf'
+include { BWAMEM2_MEM                       as BWAMEM2_MEM_POST    } from '../../../modules/nf-core/bwamem2/mem/main.nf'
+include { FGBIO_CALLMOLECULARCONSENSUSREADS as CALLUMICONSENSUS    } from '../../../modules/nf-core/fgbio/callmolecularconsensusreads/main.nf'
```

Comment:
we;ll need the addition here if sentieon for sure. I would recommend you close this PR from your own fork and instead open one from a parent sarek branch. Then the sentieon tests can run through as well.

Reply:
There are other things that I need to figure out with the central subwf, for example some inputs are annotated as [channel] but they are treated as [path]...